Cargo.toml

```diff
@@ -48,14 +48,17 @@ bitflags = "1.0"
 tokio = { version = "0.2", optional = true, features = ["macros"] }
 warp = { version = "0.2.4", optional = true }
 serde = { version = "1.0", features = ["derive"] }
+serde_bytes = "0.11"
 serde_cbor = "0.11"
-serde_json = { version = "1.0", optional = true }
+serde_json = "1.0"
 bytes = { version = "0.5", optional = true, features = ["serde"] }
-base64 = { version = "^0.10", optional = true }
+base64 = "^0.10"
+nom = "^5.1"
+sha2 = "^0.8.0"
```

Comment:
I have tried  to use the versions that are already vendored in Firefox, to avoid update procedures there that might cause problems. Currently, Firefox has sha2 in version 0.8.2. 
If you think it's not a problem to bump that dependency in Firefox as well, I can go with 0.9.